doc/GuideForCommitters.md

```diff
@@ -105,3 +105,8 @@ commits), PR builds must be triggered again.
 * Ensure `[ci skip]` is used for documentation only changes; any changes to code **must**
 be tested using the appropriate PR builds.
 
+* If the code change(s) necessitate changes to the [OpenJ9 Documentation](https://www.eclipse.org/openj9/docs/),
+ensure the contributer has an associated PR in the [openj9-docs](https://github.com/eclipse/openj9-docs) 
+repository. A PR for a code change that requires changes to the OpenJ9 Documentation 
+should not be merged until the associated PR in the openj9-docs repo is also approved 
+and ready to be merged.
```

Comment:
Can you please also add that the `depends:doc` label should be added to the OpenJ9 PR.

Reply:
Done.